jormungandr/src/settings/start/mod.rs

```diff
@@ -14,6 +14,7 @@ const DEFAULT_FILTER_LEVEL: LevelFilter = LevelFilter::TRACE;
 const DEFAULT_LOG_FORMAT: LogFormat = LogFormat::Default;
 const DEFAULT_LOG_OUTPUT: LogOutput = LogOutput::Stderr;
 const DEFAULT_NO_BLOCKCHAIN_UPDATES_WARNING_INTERVAL: u64 = 1800; // 30 min
+const DEFAULT_BLOCK_HARD_DEADLINE: u32 = 50;
```

Comment:
How did you choose this value?

Reply:
Really depends on workload and environment. For Fund-4 benchmark tallying took 180 seconds on t3a.2xlarge. This is the equivalent of 18 slots if we go with 10 seconds per slot.